Buildings/Fluid/ZoneEquipment/FanCoilUnit/Validation/FanCoilUnit_openLoop_coolingMode.mo

```diff
@@ -0,0 +1,301 @@
+within Buildings.Fluid.ZoneEquipment.FanCoilUnit.Validation;
+model FanCoilUnit_openLoop_coolingMode
```

Comment:
the name implies there will be a close-loop test. If that is not the case, let us remove the open loop